tests/lib/sprintf/src/main.c

```diff
@@ -537,6 +537,7 @@ void test_sprintf_misc(void)
 	if (IS_MINIMAL_LIBC_NANO) {
 		TC_PRINT(" MINIMAL_LIBC+CPBPRINTF skipped tests\n");
 	} else {
+#ifndef CONFIG_PICOLIBC
```

Comment:
Could you add some comments on why picolibc is excluded?

Reply:
picolibc doesn't have %n support as it is often a source of mysterious security problems when the format string is exposed to user manipulation. I'll add a comment in the test code.